Практическая работа №1.

MorzMorz opened the pull request.

Commit:
Remove classes

Commit:
Change project name

MorzMorz committed:
Практическая работа.

MorzMorz committed:
Практическая работа.
Добавлена пропущенная фигурная скобка в классе RacingCar

i-masloed commented on src/main/java/Main.java:
```diff
@@ -1,49 +1,55 @@
+import java.sql.SQLOutput;
```
Sql здесь точно не нужен, можно убрать этот импорт.
В измененных файлах стоить сделать оптимизацию импортов с помощью комбинации горячих клавиш (Control+Option+O/ Ctrl+Alt+O): это удалит все неиспользуемые импорты.

MorzMorz committed:
Практическая работа.
Добавлена проверка, чтоб пользователь вводил скорость только цифрами.
убран импорт SQLUotPut.
